src/actions/index.js

```diff
@@ -0,0 +1,59 @@
+import {
+    getStudent,
+    getStudentTotalScore
+} from '../api';
+
+export const USER_LOGIN_REQUEST = 'USER_LOGIN_REQUEST';
+export const USER_LOGIN_SUCCESS = 'USER_LOGIN_SUCCESS';
+export const USER_LOGIN_FAILURE = 'USER_LOGIN_FAILURE';
+
+export const TOTAL_SCORE_REQUEST = 'TOTAL_SCORE_REQUEST';
+export const TOTAL_SCORE_SUCCESS = 'TOTAL_SCORE_SUCCESS';
+export const TOTAL_SCORE_FAILURE = 'TOTAL_SCORE_FAILURE';
+
+export const userLoginRequest = () => ({
+    type: USER_LOGIN_REQUEST
+});
+
+export const userLoginSuccess = userData => ({
+    type: USER_LOGIN_SUCCESS,
+    userData
+});
+
+export const userLoginFailure = (login, error) => ({
```

Comment:
No need to have separate actions for everything. You can write dispatch directly in your main action creator